Makefile

```
@@ -0,0 +1,34 @@
+commit = $(shell git rev-parse HEAD)
+version = latest
+
+build: swag
+	go build .
+	@echo "Built binary found at ./wharf-provider-github or ./wharf-provider-github.exe"
+
+docker:
+	@echo docker build . \
+		-t "quay.io/iver-wharf/wharf-provider-github:latest" \
+		-t "quay.io/iver-wharf/wharf-provider-github:$(version)" \
+		--build-arg BUILD_VERSION="$(version)" \
+		--build-arg BUILD_GIT_COMMIT="$(commit)" \
+		--build-arg BUILD_DATE="$(shell date --iso-8601=seconds)"
+	@echo ""
+	@echo "Push the image by running:"
+	@echo "docker push quay.io/iver-wharf/wharf-provider-github:latest"
+ifneq "$(version)" "latest"
+	@echo "docker push quay.io/iver-wharf/wharf-provider-github:$(version)"
+endif
+
+docker-run:
+	docker run --rm -it harbor.dgc.local/tools/webhook-relay
+
+serve: swag
+	go run .
+
+swag:
+	swag init --parseDependency --parseDepth 1
```

As #16 has been merged, `--parseDepth 2` is required
```suggestion
	swag init --parseDependency --parseDepth 2
```

---

This doesn't seem to be resolved, or am I missing something?